<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="debian" failures="1" tests="1" name="Command line test" time="6.645" errors="0" timestamp="2020-10-27T16:32:26 CET">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.316">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at stepDefs.boLoginSteps.ilAccèdeAuBackOfficeMomiEtVisualiseLaListeDesMembresMomi(boLoginSteps.java:45)
at ✽.il accède au back office Momi et visualise la liste des membres Momi(file:///home/mejdi/work/personal/test-automation-framework/src/test/resources/features/bo/loginPage/login_to_backoffice.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Command line test -->
